mmpose/models/detectors/bottom_up.py

```diff
@@ -26,16 +25,14 @@ class BottomUp(BasePose):
         train_cfg (dict): Config for training. Default: None.
         test_cfg (dict): Config for testing. Default: None.
         pretrained (str): Path to the pretrained models.
-        loss_pose (dict): Config for loss. Default: None.
     """
 
     def __init__(self,
                  backbone,
                  keypoint_head=None,
                  train_cfg=None,
                  test_cfg=None,
-                 pretrained=None,
-                 loss_pose=None):
```

Comment:
For better migration experience, we can keep `loss_pose` here, and raise an error if it is not None, and give the link to this pr in the error message.